lib/manager/components/ProjectSettingsForm.js

```diff
@@ -266,20 +272,19 @@ export default class ProjectSettingsForm extends Component<Props, State> {
       )
     }
     return (
-      <div>
+      (<div>
```

Comment:
Extra parenthesis not needed.

Reply:
Prettier really struggled with these files... Fixed in d54487d0